db/seeds/development.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+##### Users #####
+
+user1 = User.create!(
+  first_name: 'Firstname1',
+  last_name: 'Lastname1',
+  email: 'user1@example.org',
+  email_confirmed: true,
+  password: 'pwd',
+  role: 'admin'
+)
+user2 = User.create!(
+  first_name: 'Firstname2',
+  last_name: 'Lastname2',
+  email: 'user2@example.org',
+  email_confirmed: true,
+  password: 'pwd'
+)
+user3 = User.create!(
+  first_name: 'Firstname3',
+  last_name: 'Lastname3',
+  email: 'user3@example.org',
+  email_confirmed: true,
+  password: 'pwd'
+)
+user4 = User.create!(
+  first_name: 'Firstname4',
+  last_name: 'Lastname4',
+  email: 'user4@example.org',
+  email_confirmed: true,
+  password: '1234'
+)
+
+AccountLink.create!(
+  user: user1,
+  name: 'My Autograder',
+  push_url: 'http://localhost:7000/import_exercise',
+  check_uuid_url: 'http://localhost:7000/import_uuid_check',
+  api_key: 'abcdefg'
+)
+
+##### Tasks #####
+
+License.create!(
+  name: 'MIT License',
+  link: 'https://opensource.org/licenses/MIT'
+)
+License.create!(
+  name: 'Creative Common Attribution',
+  link: 'https://creativecommons.org/licenses/by/4.0/'
+)
+
+Label.create!(name: 'Loops', color: '006600')
+Label.create!(name: 'Conditions', color: 'DF0101')
+Label.create!(name: 'Data Structures', color: '3333CC')
+
+pl_java = ProgrammingLanguage.create!(language: 'Java', version: '17')
+ProgrammingLanguage.create!(language: 'Python', version: '3')
+
+TestingFramework.create!(name: 'JUnit', version: '5')
+TestingFramework.create!(name: 'Pytest', version: '6')
+
+task1 = Task.create!(
+  title: 'Hello World2',
+  description: 'Write a simple program that prints "Hello World".',
+  internal_description: 'This is a simple exercise for your students to begin with Java.',
+  uuid: 'f15cb7a3-87eb-4c4c-a998-c33e25d44cdc',
+  language: 'English',
+  programming_language: pl_java,
+  user: user1
+)
+
+TaskFile.create!(
+  name: 'Main.java',
+  internal_description: 'The main java file.',
+  used_by_grader: true,
+  visible: 'yes',
+  usage_by_lms: 'edit',
+  fileable: task1,
+  content: <<~JAVA)
+    public class HelloWorld {
+      public static void main (String[] args) {
+        // System.out.println("Hello World");
+      }
+    }
+  JAVA
+
+task1_test1 = Test.create!(
+  title: 'Output Test',
+  description: 'This test checks the output.',
+  test_type: 'Unit',
+  xml_id: '123456',
+  validity: '1',
+  timeout: '30',
+  task: task1
+)
+
+TaskFile.create!(
+  name: 'HelloWorldTest.java',
+  used_by_grader: true,
+  visible: 'no',
+  usage_by_lms: 'display',
+  fileable: task1_test1,
+  content: 'TODO'
```

Comment:
@MrSerth Can you by chance please provide a JUnit snippet that tests the "Hello World" output? I am too lazy to dig back into the framework. 😜 🙈